### ISSUE TITLE
Inconsistency in how Windows platform is called in different places inside of meta.json

### ISSUE THREAD

**Issue by dsavenko** [ctuning/ck-env#30]
Hi,

In `only_for_host_os_tags` and `only_for_target_os_tags` Windows has to be defined as `windows`. In `soft_file` Windows has to be defined as `win`.

**Comment by gfursin** [comment]
Yes, I noticed that too - this is my fault (originally I was using win to be short, but then started using windows). We should agree on one and add backward compatible support to check for both "win" and "windows" ...

**Comment by dsavenko** [comment]
OK, I will provide a PR when I finish with higher priority tasks
